src/killswitch.cpp

```diff
@@ -0,0 +1,79 @@
+// #ifdef KILLSWITCH //LEAVE THIS AT THE TOP OF THIS FILE
+
+#include <Arduino.h>
+#include "killswitch.h"
+#include "ros.h"
+#include "std_msgs/Float32MultiArray.h"
+
+float currents[2];
+std_msgs::Float32MultiArray currentsMsg;
+ros::NodeHandle nh;
```

Comment:
I suggest putting to ROS node setup in main.cpp with an ifdef for rosserial. It would make development easier for electrical without having the dependence on ROS and just try keep focus on getting the IO, analog signals and embedded comms working on the boards.

Reply:
sounds good